slam-ws/src/load_waypoints/srv/WaypointRequest.srv

```diff
@@ -0,0 +1,3 @@
+int32 waypoint_number
+---
+int32[2] waypoint_coord
```

Comment:
Are we sure the waypoint coordinates are integers?

Reply:
Since there's a lot of different ways the coords could be given to us, would a string be better? https://www.ubergizmo.com/how-to/read-gps-coordinates/

Reply:
Agreed - let's just pick a convention and use that!